fdbserver/worker.actor.cpp

```diff
@@ -542,6 +542,12 @@ ACTOR Future<Void> registrationClient(Reference<AsyncVar<Optional<ClusterControl
 			}
 		} else {
 			incorrectTime = Optional<double>();
+			if (connFile->canGetFilename()) {
+				TraceEvent("ClusterFileContents")
+				    .detail("Filename", connFile->getFilename())
+				    .detail("ConnectionStringFromFile", fileConnectionString.toString())
+				    .detail("CurrentConnectionString", connectionString);
+			}
```

Comment:
For what it's worth, the connection string and the cluster file path are logged as part of `ProgramStart`. It's possible that an alternative to this would be to log a specific event whenever the cluster file is changed, though this approach might work too as long as we're certain it will be logged in a timely way whenever the file changes.

I think it may also be the case that in this block, the connection string in the file and the current connection string are the same (otherwise, you'd be in the other block).

Lastly, I think this else block might include the condition where `connFile` is null.

Reply:
Considering the connection string is not frequently changed, can we make these two events `MonitorLeaderForwarding` and `LeaderForwarding`, `trackLatest` events? So, we always know what is the current connection string that is being used by a worker/client.

Reply:
Done.